modules/install-sno_additional-requirements-for-installing-on-a-single-node-on-aws.adoc

```diff
@@ -0,0 +1,19 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_sno/install-sno-preparing-to-install-sno.adoc
+
+:_content-type: CONCEPT
+[id="additional-requirements-for-installing-on-a-single-node-on-aws_{context}"]
```

Comment:
Hello @johnwilkins, the current section structure is
```
Preparing to install on a single node
-- Additional requirements for installing on a single node on AWS  
    (Installing single-node OpenShift on AWS is mentioned in this section)

Installing OpenShift on a single node
-- Installing single-node OpenShift using the Assisted Installer
-- Installing single-node OpenShift manually
-- Creating a bootable ISO image on a USB drive
```
The _installation process_ is at the bottom of `Additional requirements ...` section, I think it's not clear enough to users, I would suggest to add a new section for AWS, e.g:
```
Preparing to install on a single node (un-touched)
-- Additional requirements for installing on a single node on AWS  
    (Installing single-node OpenShift on AWS is mentioned in this section: 
     Installing a single node cluster on AWS uses the "Installing a cluster on AWS with customizations" procedure.)

Installing OpenShift on a single node
-- Installing single-node OpenShift using the Assisted Installer
-- Installing single-node OpenShift manually
-- Installing single-node OpenShift on AWS
  ---- Requirements for installing on a single node on AWS
  ---- <Installation process>
-- Creating a bootable ISO image on a USB drive
```
WDYT？

Reply:
I created the new section for aws and put the additional requirements under there. I also created a module for installing with the process linked immediately after, since we do not incorporate xrefs into modules. I think you're right that it's clearer this way.

Reply:
@yunjiang29 add an lgtm if this is okay.